### ISSUE TITLE
Chrome: text in some edit fields appears to be missing

### ISSUE THREAD

**Issue by mehgcap** [nvaccess/nvda#8781]
### Steps to reproduce:

1. Open Chrome to a page that has a textarea element, and put a LOT of text in the field.
2. At some point, try reviewing the text. For me, NVDA reports "blank for any text after some unknown cutoff point.

### Actual behavior:

NVDA reports "blank" where the text should be. It reads normally, then cuts off partway along. I suspect a character maximum is being reached.

### Expected behavior:

NVDA should read the text as normal.

#### NVDA Installed/portable/running from source:

installed

#### NVDA version:

2018.3.1

#### Windows version:

Windows 10, v1803

#### Name and version of other software in use when reproducing the issue:

Chrome 69.0.3497.100

#### Other information about your system:

None relevant, to my knowledge

#### Does the issue still occur after restarting your PC?

unknown

#### Have you tried any other versions of NVDA?

no

**Comment by Adriani90** [comment]
@mehgcap how is the behavior in NVDA 2019.3 Beta 1 and Chrome 79?

**Comment by isidorn** [comment]
I can reproduce this also in VS Code that is built on top of Chrome.
I have latest NVDA

**Comment by isidorn** [comment]
fyi @leonardder since this issue seems rather important
Is this maybe the root cause https://bugs.chromium.org/p/chromium/issues/detail?id=1001591&q=nvda%20does%20not%20read&can=2

**Comment by isidorn** [comment]
I can not reproduce this anymore with latest Chrome.
As far as I am concerned this can be closed

**Comment by Adriani90** [comment]
Closing as works for me. Thanks for keeping us updated.
